src/credentials/auth.ts

```diff
@@ -276,7 +356,24 @@ export class Auth implements AuthService, ConnectionManager {
         return connections.find(c => c.id === connection.id)
     }
 
-    private async updateState(id: Connection['id'], connectionState: ProfileMetadata['connectionState']) {
+    private async invalidateConnection(id: Connection['id']) {
```

Comment:
A docstring explaining what this does would be helpful, because the details matter and can be tricky. E.g. does it make a backend request to invalidate, which things does it clear locally, etc.